### PR TITLE
Add netdata ports to "master-ssh" security groups

### PR DESCRIPTION
**Pull request by markbreedlove**
Add rules to allow access on Netdata's port, to the existing "master-ssh" security groups. This allows us to access Netdata via SSH tunnels.

### CODE REVIEW

**Comment by blarghmatey** [comment]
I think we also need to add that to `orchestrate/aws/mitx.sls`. That's a separate file that we use for managing settings in the edX environments.

**Comment by markbreedlove** [comment]
Yeah, just noticed that ...

File: salt/orchestrate/aws/security_groups.sls [hunk 1]
```diff
@@ -27,6 +27,13 @@ create_master_ssh_security_group:
           cidr_ip:
             - 10.0.0.0/22
             - 10.1.0.0/22
+        # netdata:
+        - ip_prodocol: tcp
+          from_port: 19999
+          to_port: 19999
+          cidr_ip:
+            - 10.0.0.0/22
+            - 10.0.0.0/22
```
Comment (blarghmatey) [inline salt/orchestrate/aws/security_groups.sls]: This has the same CIDR listed twice

Comment (markbreedlove) [reply salt/orchestrate/aws/security_groups.sls]: oops. thanks, just fixed it.

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**